packages/webdriverio/src/commands/element/click.js

```diff
@@ -37,11 +37,40 @@
  * </example>
  *
  * @alias element.click
- * @uses protocol/element, protocol/elementIdClick
+ * @uses protocol/element, protocol/elementIdClick, protocol/performActions, protocol/positionClick
  * @type action
- *
+ * @param {Object=} options object containing a property called `button` which can be set to 0 (left) 1 (middle) or 2 (right) button
  */
 
-export default function click () {
-    return this.elementClick(this.elementId)
+export default async function click (options) {
+    const { button } = options || {}
+    const isLeftButton = button === 'left' || button === 0 || !button
+    const isMiddleButton = button === 'middle' || button === 1
+    const isRightButton = button === 'right' || button === 2
+
+    if (isLeftButton) {
+        return this.elementClick(this.elementId)
+    }
+
+    if (isRightButton || isMiddleButton) {
+        if (this.isW3C) {
```

Comment:
Will you be able to add devtools protocol support or you want to deal with it with another ticket?

Reply:
Good point, since I will be leaving for my vacation tomorrow I think it's best to do it in a separate ticket :)

Reply:
I'll raise a ticket, have a great vacation